inc/app_replacer.php

```diff
@@ -203,7 +203,7 @@ public static function possible_data_ignore_flags() {
 	 * @return array Size mapping.
 	 */
 	protected static function size_to_crop() {
-		if ( null !== self::$size_to_crop && is_array( self::$size_to_crop ) ) {
+		if ( ! empty( self::$size_to_crop ) && is_array( self::$size_to_crop ) ) {
```

Comment:
Can you also change similar occurrences in this file so that we have consistency.
The `null !== ` check might cause issues with cases that we do not test.
Also make sure to also check the #302 PR for the same changes.